docs/intro/installation.rst

```diff
@@ -4,84 +4,41 @@
 Installation
 ************
 
-The SunPy project `maintains a range of packages <https://sunpy.org/project/affiliated>`__ that leverage the wider ecosystem of scientific Python packages for solar physics.
+This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and sunpy.
 
 Installing Python
 =================
-These instructions will set you up with `miniforge <https://conda-forge.org/docs/user/introduction.html>`__, which makes it easy to install and manage Python packages.
+There are many ways to install Python, but even if you have Python installed somewhere on your computer we recommend following these instructions anyway.
+That's because we will create a new Python environment.
+As well as containing a Python installation, this environment provides an isolated place to install Python packages (like sunpy).
```

Comment:
```suggestion
That's because we will create a new Python environment that is isolated place to install Python packages (like ``sunpy``) without affecting any current Python installation.
```